sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -46,19 +46,21 @@ export class AssessmentControllerService {
    * @param assessmentId assessment id to be retrieved.
    * @param options for NOA.
    * - `studentId` optional student for authorization when needed.
+   * - `applicationId` application id,
    * - `maskMSFAA` mask MSFAA or not.
    * @returns notice of assessment data.
    */
   async getAssessmentNOA(
     assessmentId: number,
     options?: {
       studentId?: number;
+      applicationId?: number;
       maskMSFAA?: boolean;
     },
   ): Promise<AssessmentNOAAPIOutDTO> {
     const assessment = await this.assessmentService.getAssessmentForNOA(
       assessmentId,
-      options?.studentId,
+      { studentId: options?.studentId, applicationId: options?.applicationId },
```

Comment:
Just to raise a point that was a subject of a conversation with @dheepak-aot, due to the fact the `applicationId` was authorized by the decorator, we no longer need to include the `studentId` in the queries unless required for some other reason than authorization.